lib/bootsnap/setup.rb

```diff
@@ -24,12 +24,15 @@
   cache_dir = File.join(app_root, 'tmp', 'cache')
 end
 
+ruby_version = Gem::Version.new(RUBY_VERSION)
+iseq_cache_enabled = ruby_version < Gem::Version.new('2.5.0') || ruby_version >= Gem::Version.new('2.6.0')
```

Comment:
Just for my own curiosity, why are these matchers more correct versus something like `RUBY_VERSION !~ /\A2\.5/`?

Reply:
In this case they are not, I was just following the same pattern other parts of the codebase was using.

Reply:
Ah, I see. All good 👍